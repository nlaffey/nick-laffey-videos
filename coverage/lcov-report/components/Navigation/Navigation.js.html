<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for components/Navigation/Navigation.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../prettify.css" />
    <link rel="stylesheet" href="../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../index.html">All files</a> / <a href="index.html">components/Navigation</a> Navigation.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>1/1</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import {Link} from 'react-router';
import VideoInfo from '../Videos/VideoInfo';
class Navigation extends React.Component {
    render() {
        return &lt;div className='nav'&gt;
            &lt;div className='nav-left'&gt;
                &lt;div className="nav-header"&gt;&lt;Link to="home"&gt;NICK LAFFEY&lt;/Link&gt;&lt;/div&gt;
                &lt;a className='hover-bold' href="mailto:nicklaffey@gmail.com"&gt;NICKLAFFEY@GMAIL.COM&lt;/a&gt;&lt;/div&gt;
            &lt;div className="nav-center"&gt;
                &lt;VideoInfo videoData={this.props.videoData}/&gt;
            &lt;/div&gt;
            &lt;div className='nav-right'&gt;
                &lt;div className="nav-header"&gt;VIDEOS&lt;/div&gt;
                &lt;ul&gt;
                    &lt;li className='hover-bold'&gt;&lt;Link to="midtowngreenway"&gt;MIDTOWN
                        GREENWAY&lt;/Link&gt;&lt;/li&gt;
                    &lt;li className='hover-bold'&gt;&lt;Link to="letitdie"&gt;LET IT DIE&lt;/Link&gt;&lt;/li&gt;
                &lt;/ul&gt;
            &lt;/div&gt;
&nbsp;
        &lt;/div&gt;
    }
}
&nbsp;
export default Navigation;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Mon Sep 19 2016 22:17:58 GMT-0400 (EDT)
</div>
</div>
<script src="../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../sorter.js"></script>
</body>
</html>
