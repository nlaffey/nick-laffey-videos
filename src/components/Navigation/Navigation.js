import React from 'react';
import {Link} from 'react-router';
import VideoInfo from '../Videos/VideoInfo';
class Navigation extends React.Component {
    render() {
        return <div className='nav'>
            <div className='nav-left'>
                <div className="nav-header"><Link to="home">NICK LAFFEY</Link></div>
                <a className='hover-bold' href="mailto:nicklaffey@gmail.com">NICKLAFFEY@GMAIL.COM</a></div>
            <div className="nav-center">
                <VideoInfo videoData={this.props.videoData}/>
            </div>
            <div className='nav-right'>
                <div className="nav-header">VIDEOS</div>
                <ul>
                    <li className='hover-bold'><Link to="midtowngreenway">MIDTOWN
                        GREENWAY</Link></li>
                    <li className='hover-bold'><Link to="letitdie">LET IT DIE</Link></li>
                </ul>
            </div>

        </div>
    }
}

export default Navigation;