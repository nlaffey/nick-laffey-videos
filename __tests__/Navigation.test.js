import renderer from 'react-test-renderer';
import React from 'react';
import Navigation from '../src/components/Navigation/Navigation';
import LetItDieImage from '../src/img/letitdie.png';


it('renders correctly 1', ()=> {
    const videoData = {
        name: "Let It Die",
        image: LetItDieImage,
        location: "Shot in the Raring Building at the University of Minnesota",
        href: "https://www.youtube.com/watch?v=-0xqFsr1b0k",
        date: "April 2013",
        videoBy: "Meagan Nouis",
        musicBy: "Fiest"
    }
    const video = renderer.create(
        <Navigation videoData={videoData}/>
    ).toJSON();
    expect(video).toMatchSnapshot();
});
