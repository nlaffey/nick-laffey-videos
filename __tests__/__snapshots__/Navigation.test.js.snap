exports[`test renders correctly 1 1`] = `
<div
  className="nav">
  <div
    className="nav-left">
    <div
      className="nav-header">
      <a
        onClick={[Function bound handleClick]}
        style={Object {}}>
        NICK LAFFEY
      </a>
    </div>
    <a
      className="hover-bold"
      href="mailto:nicklaffey@gmail.com">
      NICKLAFFEY@GMAIL.COM
    </a>
  </div>
  <div
    className="nav-center">
    <div
      className="info">
      <ul>
        <li>
          <strong />
        </li>
        <li>
          April 2013
        </li>
        <li>
          Shot in the Raring Building at the University of Minnesota
        </li>
        <li>
          Video by 
          Meagan Nouis
        </li>
        <li>
          Music by 
          Fiest
        </li>
      </ul>
    </div>
  </div>
  <div
    className="nav-right">
    <div
      className="nav-header">
      VIDEOS
    </div>
    <ul>
      <li
        className="hover-bold">
        <a
          onClick={[Function bound handleClick]}
          style={Object {}}>
          MIDTOWN GREENWAY
        </a>
      </li>
      <li
        className="hover-bold">
        <a
          onClick={[Function bound handleClick]}
          style={Object {}}>
          LET IT DIE
        </a>
      </li>
    </ul>
  </div>
</div>
`;
